sql-ballerina/Module.md

```diff
@@ -0,0 +1,371 @@
+sql:IntegerValue age = new (10);
+
+sql:ParameterizedQuery query = `INSERT INTO student(age, name)
+                                values (${age}, ${name})`;
+var ret = dbClient->execute(query);
+f (ret is sql:ExecutionResult) {
+    io:println("Inserted row count to Students table: ", ret.affectedRowCount);
+} else {
+    error err = ret;
+    io:println("Insert to Students table failed: ", err.message());
+}
+```
+
+#### Inserting data with auto-generated keys
+
+This example demonstrates inserting data while returning the auto-generated keys. It achieves this by using the 
+`execute` remote function to execute the INSERT statement.
+
+```ballerina
+int age = 31;
+string name = "Kate";
+
+sql:ParameterizedQuery query = `INSERT INTO student(age, name)
+                                values (${age}, ${name})`;
+var ret = dbClient->execute(query);
+if (ret is sql:ExecutionResult) {
+    int? count = ret.affectedRowCount;
+    string|int? generatedKey = ret.lastInsertId;
+    io:println("Inserted row count: ", count);
+    io:println("Generated key: ", generatedKey);
+} else {
+    error err = ret;
+    io:println("Insert to table failed: ", err.message());
+}
+```
+
+#### Querying data
+
+This sample shows three examples to demonstrate the different usages of the `query` operation and query the
+database table and obtain the results. 
+
+This example demonstrates querying data from a table in a database. 
+First, a type is created to represent the returned result set. Note the mapping of the database column 
+to the returned record's property is case-insensitive (i.e., `ID` column in the result can be mapped to the `id` 
+property in the record). Next, the SELECT query is executed via the `query` remote function of the client by passing that 
+result set type. Once the query is executed, each data record can be retrieved by looping the result set. The `stream` 
+returned by the select operation holds a pointer to the actual data in the database and it loads data from the table 
+only when it is accessed. This stream can be iterated only once. 
+
+```ballerina
+// Define a type to represent the results.
+type Student record {
+    int id;
+    int age;
+    string name;
+};
+
+// Select the data from the database table. The query parameters are passed 
+// directly. Similar to the `execute` examples, parameters can be passed as
+// sub types of `sql:TypedValue` as well.
+int id = 10;
+int age = 12;
+sql:ParameterizedQuery query = `SELECT * FROM students
+                                WHERE id < ${id} AND age > ${age}`;
+stream<Student, sql:Error> resultStream = 
+        <stream<Student, sql:Error>> dbClient->query(query, Student);
+
+// Iterating the returned table.
+error? e = resultStream.forEach(function(Student student) {
+   io:println("Student Id: ", student.id);
+   io:println("Student age: ", student.age);
+   io:println("Student name: ", student.name);
+});
+if (e is error) {
+   io:println("Query execution failed.", e);
+}
```

Comment:
don't need this code here

Reply:
please check all places

Reply:
removed in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/d3df4347c964f2a634909920ca548fefdfc1139c